mote/.stylelintrc

```diff
@@ -0,0 +1,359 @@
+    rule-nested-empty-line-before:
+        - always-multi-line
+        - except:
+            - first-nested
+        - ignore:
+            - after-comment
+    rule-non-nested-empty-line-before:
+        - always
+        - ignore:
+            - after-comment
+
+    # Media feature (http://stylelint.io/user-guide/rules/#media-feature)
+    media-feature-colon-space-after: always
+    media-feature-colon-space-before: never
+    media-feature-name-case: lower
+    media-feature-name-no-unknown: true
+    media-feature-name-no-vendor-prefix: true
+    media-feature-no-missing-punctuation: true
+    media-feature-parentheses-space-inside: never
+    media-feature-range-operator-space-after: always
+    media-feature-range-operator-space-before: always
+
+    # Media query list (http://stylelint.io/user-guide/rules/#media-query-list)
+    media-query-list-comma-newline-after: always-multi-line
+    media-query-list-comma-newline-before: never-multi-line
+    media-query-list-comma-space-after: always
+    media-query-list-comma-space-before: never
+
+    # At-rule (http://stylelint.io/user-guide/rules/#at-rule)
+    # at-rule-empty-line-before:
+    #     - always
+    #     - ignore:
+    #         - after-comment
+    #     - except:
+    #         - blockless-group
+    #         - blockless-after-same-name-blockless
+    #     - ignoreAtRules:
+    #         - if
+    #         - else
+    at-rule-name-case: lower
+    at-rule-name-space-after: always
+    at-rule-no-vendor-prefix: true
+    at-rule-semicolon-newline-after: always
+
+    # stylelint-disable comment (http://stylelint.io/user-guide/rules/#stylelint-disable-comment)
+    stylelint-disable-reason: always-after
+
+    # Comment (http://stylelint.io/user-guide/rules/#comment)
+    # comment-empty-line-before:
+    #     - always
+    #     - except:
+    #         - first-nested
+    #     - ignore:
+    #         - between-comments
+    #         - stylelint-commands
+    comment-no-empty: true
+    comment-whitespace-inside: always
+
+    # General / Sheet (http://stylelint.io/user-guide/rules/#general--sheet)
+    indentation: 4
+    max-empty-lines: 5
+    max-line-length: 160
+    max-nesting-depth: 4
+    no-duplicate-selectors: true
+    no-eol-whitespace:
+        - true
+        - ignore:
+            - empty-lines
+    no-extra-semicolons: true
+    no-invalid-double-slash-comments: true
+    no-missing-end-of-source-newline: true
+
+    # stylelint-scss (https://www.npmjs.com/package/stylelint-scss)
+    scss/at-extend-no-missing-placeholder: true
+    scss/at-import-no-partial-leading-underscore: true
+    scss/at-mixin-argumentless-call-parentheses: never
+    scss/dollar-variable-colon-space-after: always
+    scss/dollar-variable-colon-space-before: never
+    scss/operator-no-newline-after: true
+    scss/operator-no-newline-before: true
+    scss/operator-no-unspaced: true
+    scss/selector-no-redundant-nesting-selector: true
```

Comment:
from what I understood it looks decent.

one question is there a way too specify that we want all nested blocks and mixins to come after properties in a block? we use to have this in sasslint